### DISCUSSION TITLE
Low signal problem

### DISCUSSION THREAD

**Discussion by alex8654** [Koenkk/zigbee2mqtt#15093] [General]
I've got pretty rubbish signal on my zigbee devices and I was wondering if you guys can share some ideas please. I've got a couple of https://smlight.tech/product/slzb-06/, one in a garage the other one in a house, in theory I should get pretty good signal, but even with my devices nearly stick to the antenna, the best I can get is 170 LQI. If the device is 5m away, behind a wall made of plasterboard, the signal drops below 60. I've disabled my WIFI in the garage to avoid any 2.4ghz interference, and the result was the same. The garage has pretty thick walls, barely any 2.4ghz noise from other networks.

I've also tried different channels, starting on 11, all the way to 26, still same LQI, at best 175. I see many screne shots of people getting 255 LQI o many devices, so I cannot understand why my coordinators that are using a network cable and at excellent locations away from computers and other interference sources to have such a bad signal.

Has anyone got any tips of what else I can try?

**Comment by lux73** [comment]
First: all LQI above 20 are considered as good - there are no expected Problems with such an "low" LQI

so many People have an Bunch of Powered Zigbee Devices (those works as Router) & some Battery Devices would connect to the nearest Router

**please Note: the LQI shown in the Network Map has nothing to do with LQI showing in the Webinterface**

if all your Devices are above LQI 20 you should be good

**Comment by alex8654** [comment]
Ok, so after doing a bit more research and understanding better how signal in a zigbee network works, I know understand the LQI numbers don't really reflect the actual experience. I ended up putting the devices all back to normal settings, on channels 11 and 25, it is working fine and I can get zigbee devices working on 2 different buildings controlled in a single home assistance instance, pretty cool.

Even devices with 30LQI are working fine reporting back with no delay, so it was my miss understand of how it works. I just wish there would be some documentation explaining this better. There is nothing on Z2M to educate end users about acceptable signal levels and the difference between different chipsets reporting different signal numbers when testing the same device.

Anyway, another thing to learn.

**Comment by cleversmart** [comment]
But also for my understanding, please explain me this behavior:
Same environment with conbee II = LQI 230-255, with SLZB-06 = LQI 35-120
why is this so different?
the response is better with conbee II in the network

**Reply by OddyAte** [reply]
I am in the same situation, had a Conbee II and expected way better LQI from the SLZB-06 because of the antenna but it's not the case.

**Reply by chris-1243** [reply]
The LQI shown in the GUI is not always accurate. I do not remember where I read this but it is known LQI in `deconz` is not the most accurate. This information is provided by the firmware if I am not wrong... so if you move to `ember`, Silabs chip, you will different values for the same network.

Have a look at your network map where you will see the multiple link and those values should be correct.

When all your LQI value are above 230 even for device far from the coordinator, that sounds strange...
